Microsoft.Alm.Authentication/Secret.cs

```diff
@@ -54,6 +54,22 @@ public static string UriToUrl(TargetUri targetUri, string @namespace)
             return targetName;
         }
 
+        /// <summary>
+        ///     Generate a key based on the ActualUri.
+        ///     This may include username, port, etc
+        /// </summary>
+        public static string UriToActualUri(TargetUri targetUri, string @namespace)
```

Comment:
nit: Use `To*Uri` method names when returning a `Uri`; use `To*Url` method names when returning a URL formatted `string`.